hooks/blueprint

```diff
@@ -13,10 +13,14 @@ validate_features nfs-volume-services small-footprint \
                   minimum-vms \
                   autoscaler autoscaler-postgres \
                   haproxy-tls haproxy-notls omit-haproxy \
-                  migrate-1.3-without-consul
+                  migrate-1.3-without-consul\
+                  container-routing-integrity\
+                  native-garden-runc\
```

Comment:
Please put spaces before the trailing `\` line continuation character

Reply:
Done